inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/util/IpPort.java

```diff
@@ -19,16 +19,20 @@
 
 import io.netty.channel.Channel;
 import org.apache.commons.lang3.math.NumberUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.net.InetSocketAddress;
 
 public class IpPort {
```

Comment:
ditto, ipport is not a word, how about using "address" or other words?

Reply:
Fixed